src/main/java/de/geolykt/enchantments_plus/PermissionHandler.java

```diff
@@ -0,0 +1,22 @@
+package de.geolykt.enchantments_plus;
+
+import org.bukkit.command.CommandSender;
+
+public class PermissionHandler {
+
+
+    public static boolean hasPermission(EnchantPlayer player, PermissionTypes permissionType) {
+        return PermissionHandler.hasPermission((CommandSender)player, permissionType);
```

Comment:
This will NOT work and throw a Class cass exception, unless I misunderstand something

Reply:
You're right, was missing the `player.getPlayer()` to get the `Player` from the `EnchantPlayer` object